### DISCUSSION TITLE
How about if we can customlize the hashtag definition to determine the output directory ?

### DISCUSSION THREAD

**Discussion by CaiQi1994** [etherealxx/batchlinks-webui#13] [Ideas]
As a colab user ,each time launch the program I will download some temp models and output it to directory out of my google drive . and then make a soft link to this directory from model reading directory .
I make above with notebook command. So if this extension can customlize hashtags , for example,to definite a hashtag #temp_model for /content/temp_model as output dir, will be useful for me .

**Comment by etherealxx** [comment]
Thank you for the suggestion! ✨That kind of feature is already on my roadmap. Currently I'm thinking about the best way to input the custom hashtag and the path (like the syntax of it). 

I need to make sure it is easy to understand for beginners and not that confusing.
Maybe something like
```
@new #temp_model /content/temp_model
```
Or maybe
```
@#temp_model /content/temp_model
```
Something like that😁

I could, like, add a drop-down menu or more buttons/textbox, but that goes against my philosophy towards this extension, i want it to be simple and not bloated.

Currently I'm working on the next major update. I'll reach out to you again once I have a progress😉

**Comment by etherealxx** [comment]
Hello👋 The update is now live! i added that feature: https://github.com/etherealxx/batchlinks-webui#custom-hashtag-path-new
